designdocs/logical_batching.md

```diff
@@ -0,0 +1,67 @@
+# Summary
+
+In other documents, we explore ways to structure an explicit batch of messages.
+That is, in terms of concrete structure. This document has a different aim --
+deriving the idea of a batch logically, as a product of fields present on other
+entities in our system.
+
+## Background
+
+We are currently experimenting with a new system architecture that has the
+following entities:
+
+1. Schema
+   - Id
```

Comment:
All these fields need types.  Also, are we going to have a "pay more" type of field? We were originally going to use the file size but @wilwade suggested it be something more general. Don't know what it would be called.

Reply:
Re: pay more / files size

I think we should stick with `payload_size` (perhaps `payload_size_bytes`?) for now, although for clarity that goes under message (line 18/19) not Schema (which is where some form of pay more setup might go)